src/layouts/SecondaryLayout/SecondaryLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import { Header } from '@/components';
+import { IHeader } from '@/components/Header/types';
+import { Button } from '@/components/ui/button';
+import { ChevronLeft } from 'lucide-react';
+
+type ISecondaryLayout = {
+  header: Pick<IHeader, 'title'>;
+  onBackClick?: () => void;
+  children: React.ReactNode;
```

Comment:
Se children for opcional, pode utilizar o `PropsWithChildren` ao invés de redeclarar o children.
Na linha 16 ficaria `}: PropsWithChildren<ISecondaryLayout>) => {`